src/components/manage/index.js

```diff
@@ -119,11 +119,45 @@ export class Manage extends Component {
     e.stopPropagation()
 
     if (!this.state.formPristine) {
-      console.log('continue with saving...')
-      return
-    }
+      const { crowdsaleStore, tierStore } = this.props
+      const updatableTiers = crowdsaleStore.selected.initialTiersValues.slice().filter(tier => tier.updatable)
```

Comment:
`.slice()` is not necessary here, `.filter` does not mutate the underlying array.

Reply:
fixed